npc/quests/quests_umbala.txt

```diff
@@ -1003,7 +1003,7 @@ um_in,139,48,5	script	Phrenetan	783,{
 		close;
 	}
 	if (um_wind <= 3 && (MISC_QUEST & 32768) == 0) {
-		if (um_wind) set um_wind,1;
+		if (um_wind == 0) set um_wind,1;
```

Comment:
I'd rather use `!um_wind`

Reply:
Well the two formats for this issue are corrects, it's cosmetic

Reply:
I know, it's just my OCD lol